src/cmdline.rs

```diff
@@ -416,6 +422,13 @@ pub fn parse_args() -> Result<Config> {
                                 .help("Description of OS")
                                 .takes_value(true),
                         )
+                        .arg(
+                            Arg::with_name("real-rootdev")
```

Comment:
Hmm, why not just have coreos-installer auto-detect this and open up the LUKS container itself?

Reply:
Ahh OK I think I follow now. You already have the data offset from `rhcos_luks_header_size=32768`. Neat.

Hmm, is that offset in sector size? I think it might need a `* 512` before adding to the fiemap offsets.

Reply:
Dunno; setting up device-mapper feels like it's something that should live externally?  Dunno.

Reply:
> Hmm, is that offset in sector size? I think it might need a * 512 before adding to the fiemap offsets.

Ooh yes!  Of course.  And that matches up with
```
Data segments:
  0: crypt
	offset: 16777216 [bytes]
```

Got a successful osmet checksum match, doing a full `cosa buildextend-live` now!

Reply:
```
Successfully tested scenario iso-offline-install for 46.82.202005261954-0 on bios (metal)
```

:tada:

Reply:
Random aside, a full 16MiB for the LUKS header seems *enormous*...not sure what's going on there.  That was one of the things that made me think the value was in bytes.